.coveragerc

```diff
@@ -9,6 +9,11 @@ source =
 skip_covered = true
 skip_empty = true
 show_missing = true
+exclude_also =
+    if TYPE_CHECKING
+    assert False
```

Comment:
Where have you seen this one?

Reply:
I copied the block from `aiohttp`.  We don't need it here.  I'll clean it up once the CI passes

Reply:
I'm replacing it in #1085. You can just drop it then.